docs/docsite/rst/dev_guide/platforms/openstack_guidelines.rst

```diff
@@ -1,60 +0,0 @@
-.. _OpenStack_module_development:
```

Comment:
@acozine @abadger Should dev_guide documentation guides stay in ansible/ansble?
@sshnaidm Does this documentation exist elsewhere?

Reply:
@gundalow not in openstack repo, but as I see it's mostly incorrect atm. So anyways we'll need to write a new one. I'll submit a patch with this file and we'll start to review it. In the current state it's totally outdated.

Reply:
@gundalow submitted in https://review.opendev.org/#/c/704558/

Reply:
Perfect, thank you.